### PR TITLE
Fix some errors

### PR DESCRIPTION
**Pull request by szepeviktor**
@seb86 If you want to discover bugs, please use PHPStan.

### CODE REVIEW

File: includes/classes/utilities/class-cocart-utilities-cart-helpers.php [hunk 1]
```diff
@@ -941,8 +941,8 @@ public static function has_enough_stock( $current_data = array(), $quantity = 1
 					/* translators: 1: Quantity Requested, 2: Product Name, 3: Quantity in Stock */
 					__( 'You cannot add that amount of (%1$s) for "%2$s" to the cart because there is not enough stock, only (%3$s remaining).', 'cart-rest-api-for-woocommerce' ),
 					$quantity,
-					$product->get_name(),
-					wc_format_stock_quantity_for_display( $stock_quantity, $product )
+					$current_product->get_name(),
+					wc_format_stock_quantity_for_display( $stock_quantity, $current_product )
```
Comment (szepeviktor) [inline includes/classes/utilities/class-cocart-utilities-cart-helpers.php:945]: undefined variable

Comment (seb86) [reply includes/classes/utilities/class-cocart-utilities-cart-helpers.php:945]: Good find

**Comment by seb86** [comment]
@szepeviktor Yeah I will when I get round to it. Thanks for finding them for me.

**Comment by seb86** [comment]
Have I setup PHPStan correctly @szepeviktor ? Created [new branch](https://github.com/co-cart/co-cart/tree/trunk-phpstan) to test on.

**Comment by szepeviktor** [comment]
I used these 3 packages
php-stubs/woocommerce-stubs php-stubs/wp-cli-stubs szepeviktor/phpstan-wordpress
and this config
```neon
parameters:
    paths:
        - cart-rest-api-for-woocommerce.php
        - load-package.php
        - uninstall.php
        - includes/
    bootstrapFiles:
        - vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php
    scanFiles:
        - vendor/php-stubs/wp-cli-stubs/wp-cli-stubs.php
    level: 0
```

**Pull request merged**